<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QPushButton" name="startButton">
      <property name="text">
       <string>Start(&amp;F)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectDirButton">
      <property name="text">
       <string>选择文件夹</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectFileButton">
      <property name="text">
       <string>选择文件</string>
      </property>
     </widget>
     <widget class="QPushButton" name="configButton">
      <property name="text">
       <string>配置文件</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
